components/ui-shell/.changeset/top_navbar_197.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
This file should be on `.changeset` on the root folder. Could you move it there so the bot can detect the changeset?

Reply:
thanks! fixed in https://github.com/Qiskit/web-components/pull/197/commits/37b2c68bd96158d6be7ed0cf2ee5e6828807811f